gcc/config/arc/arc.md

```diff
@@ -2271,7 +2271,7 @@ archs4x, archs4xd"
       && !satisfies_constraint_L (operands[1]))
     {
       /* MUL64 <0,>b,s12 00101bbb10000100 0BBBssssssSSSSSS  */
-      int n = true_regnum (operands[0]);
+      int n = DBX_REGISTER_NUMBER(true_regnum (operands[0]));
```

Comment:
@claziss how do you think, do we need to apply `DBX_REGISTER_NUMBER` to all other places where `true_regnum` is used?